### PR TITLE
Enable Certificate Authentication for Geneva Agent

### PR DESCRIPTION
**Pull request by RyoYang**
1. Support certificate type for geneva agent authentication 
2. Documentation enhancement and typo fix

### CODE REVIEW

File: moneo.py [hunk 1]
```diff
@@ -146,9 +146,15 @@ def deploy_worker(self, hosts_file, max_threads=16):
             cmd = cmd + ' false'
         if self.args.launch_publisher:
             agent = self.args.launch_publisher
-            print('-Enable ' + agent + ' agent-')
-            logging.info('Enable ' + agent + ' agent')
             cmd = cmd + ' ' + agent
+            if self.args.publisher_auth:
```
Comment (rafsalas19) [inline moneo.py:150]: Whats the functionality if a user selects to use the publisher and doesn't provide the auth argument?

File: moneo.py [hunk 2]
```diff
@@ -354,6 +360,12 @@ def parallel_ssh_check():
         '--launch_publisher',
         type=str,
         help='This launches the publisher which will share exporter data with Azure. Choices: {geneva, azure_monitor}.')
+    parser.add_argument(
+        '-a',
+        '--publisher_auth',
+        type=str,
+        help='Required if launching publisher with geneva. Authentication method for geneva. Choices: {umi, cert}.'
+             'Please replace the mdm-key.pem and mdm-cert.pem in src/worker/publisher/config with yours if using cert.')
```
Comment (rafsalas19) [inline moneo.py:368]: Minor formatting: place a space between ```Choices: {umi, cert}.Please replace```

File: moneo.py (hunk 1 above)
Comment (RyoYang) [reply moneo.py:160]: Thanks for catching, addressed.

**Comment by rafsalas19** [comment]
Can you make one more addition in Moneo/linux_service/start_moneo_services.sh line 14, can you add the 'cert argument'.

i.e. 
```$MONEO_PATH/src/worker/start_geneva.sh  cert  $MONEO_PATH/src/worker/publisher/config/geneva_config.json```

I think this part is fine to hard code. I will make the necessary changes in my docs pr for all the additions added by this pr.

**Comment by RyoYang** [comment]
> Can you make one more addition in Moneo/linux_service/start_moneo_services.sh line 14, can you add the 'cert argument'.
> 
> i.e. `$MONEO_PATH/src/worker/start_geneva.sh cert $MONEO_PATH/src/worker/publisher/config/geneva_config.json`
> 
> I think this part is fine to hard code. I will make the necessary changes in my docs pr for all the additions added by this pr.

Done.

**Review by rafsalas19 — APPROVED** [review]

**Pull request merged**